Add instructions for Power BI connector (#1079)
spiceai/docs: website/docs/clients/powerbi/index.md

---

> title: 'Power BI'

Reviewer: ```suggestion
title: 'Microsoft Power BI Connector'
```

> sidebar_label: 'Power BI'

Reviewer: ```suggestion
sidebar_label: 'Microsoft Power BI'
```

sidebar_position: 11

> description: 'Use Microsoft Power BI to access, visualise and analyse datasets loaded in Spice.'

Reviewer: ```suggestion
description: 'Use Microsoft Power BI to access, visualize and analyze Spice datasets.'
```

pagination_prev: 'clients/index'
pagination_next: null
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Use the instructions below to get started with the **[Spice.ai Power BI Connector](https://github.com/spiceai/powerbi-connector)**—an [ADBC](https://github.com/apache/arrow-adbc)-based connector that enables [Microsoft Power BI](https://www.microsoft.com/en-us/power-platform/products/power-bi) users to easily connect to and visualize data loaded in [Spice.ai Enterprise](https://spiceai.org/) and [Spice Cloud Platform](https://spice.ai/) instances.

## Manual Connector Installation

> ### Tableau Desktop

Reviewer: ```suggestion
### Power BI Desktop
```
